scripts/decl-name-changes.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -e
```

Perhaps make this script exit on error. But then again the files declsOld.txt declsNew.txt might not be cleaned up in that case...

Reply:
I haven't made it exit on error, but I have added a warning to the usage that early termination could cause git to end up in a detached head state.